packages/transformers/elm/src/ElmTransformer.js

```diff
@@ -0,0 +1,128 @@
+// @flow
+import {Transformer} from '@parcel/plugin';
+import commandExists from 'command-exists';
+import path from 'path';
+import fs from 'fs';
+import spawn from 'cross-spawn';
+import elm from 'node-elm-compiler';
+import {inject as injectElmHMR} from 'elm-hot';
+import {minify} from 'terser';
+import ThrowableDiagnostic from '@parcel/diagnostic';
+
+export default new Transformer({
+  async transform({asset, options}) {
+    const config = {
+      cwd: path.dirname(asset.filePath),
+      debug: !(
+        options.env.PARCEL_ELM_NO_DEBUG || options.mode === 'production'
```

Comment:
No, environment variables aren't factored into the cache key. So this would require you to delete the cache directory when changing that variable.

Can this somehow be read from elm.json?

Reply:
Didn't know that `options.env` is cached. Elm doens't allow third party options in `elm.json`.
Is `process.env` accessable in Transformers?

Reply:
No, using `options.env` is correct.
The problem is that the resolver just doesn't rerun at all if only the environment variable changes. But this is really a bug : https://github.com/parcel-bundler/parcel/issues/4264. I guess it's fine for now..